src/util/Handlers.js

```diff
@@ -0,0 +1,47 @@
+/*!
+ * Copyright (c) 2015-2016, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+define(['util/Logger'], function (Logger) {
+
+  var Handlers = {};
+
+  Handlers.defaultSuccessTokenHandler = function(tokenManager) {
```

Comment:
Open question - do we make the key values configurable? I personally like the idea of hardcoding these to `idToken` and `accessToken`, as it'll align with our README documentation and samples.